docs/projects/variables/aws-account-variables.md

```diff
@@ -40,6 +40,11 @@ Write-Host 'AwsAccount.Id=' #{aws account}
 Write-Host 'AwsAccount.AccessKey=' #{aws account.AccessKey}
 ```
 
+:::hint
+**Parameter naming convention**
+The name of the Octopus Parameter will be mapped to the name of the variable that was defined. As shown in the PowerShell example, one of the parameters is named "aws account" because the variable defined above was named "AWS account". If the variable were named "Test Account Information", the PowerShell example would need to be changed to "test account information".
```

Comment:
```suggestion
The name of the Octopus Parameter will be mapped to the name of the variable that was defined. As shown in the PowerShell example, one of the parameters is named `aws account` because the variable defined above was also named `aws account`. If the variable were named `test account information`, the PowerShell example would need to be changed to `test account information.
```